CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## To be Released
 
+* chore(go): use go 1.23
+* Raise default length from 20 to 24
```

Comment:
question: is this new default length (24) the minimum required by our PSSI?

edit: after reading the related story, I think the default length should be the _maximum_ length. Hence it should be 64. What do you think?